index.js

```diff
@@ -27,18 +27,18 @@ module.exports = function(sass) {
     this.inputFile = inputFile;
     this.outputFile = outputFile;
 
-    this.renderSass = rsvp.denodeify(sass.render);
+    this.renderSass = sass.compileAsync;
```

Comment:
Can you please explain inline what each of these changes does?

Reply:
This is now legacy code: https://sass-lang.com/documentation/js-api/modules#render
The new way is with
https://sass-lang.com/documentation/js-api/modules#compileAsync